application.py

```diff
@@ -33,3 +33,12 @@
 application = Flask('app')
 application.wsgi_app = ProxyFix(application.wsgi_app)
 create_app(application)
+
+if os.environ.get('USE_LOCAL_JINJA_TEMPLATES') == 'True':
+    print('')
```

Comment:
Maybe add 

print(.env `USE_LOCAL_JINJA_TEMPLATES=True`)

As a pointer back to that file